src/Controller/Ssofact.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Drupal\ssofact\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+
+class Ssofact extends ControllerBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function content() {
+    return [];
```

Comment:
Would be great if we could add some kind of a placeholder string here, so that the SSO developers can inject their HTML output more easily; for example:

```
{{page_content}}
```
Ideally do the same for the page title.